velox/exec/GroupingSet.cpp

```diff
@@ -286,6 +286,8 @@ void GroupingSet::initializeGlobalAggregation() {
 
   for (auto& aggregate : aggregates_) {
     aggregate->setAllocator(&stringAllocator_);
+    // Accumulators must be byte-aligned by their fixed width size.
+    offset = bits::roundUp(offset, aggregate->accumulatorFixedWidthSize());
```

Comment:
Align to 8 bytes should be enough.  For some larger object, this could waste too much memory.

Reply:
@Yuhta, For LongDecimal (int128_t), we need 16 bytes. For structs, I see your point. I believe we must align by the size of the largest alignment required among the elements.

Reply:
Let's stick to 16 bytes. I will add a comment.

Reply:
If this causes some memory issue in the future, we can add one more virtual method to aggregate to return different alignment per accumulator type.